ui/src/lib/components/Form.svelte

```diff
@@ -51,41 +56,43 @@
             <input
                 type="text"
                 on:input={handlePathInput}
-                bind:value={formData.pathText}
+                bind:value={formData.path}
                 placeholder="C:/images"
+                autocomplete="on"
             />
         </label>
-        {#if wrongPath && formData.pathText !== ''}
+        {#if wrongPath && formData.path !== ''}
```

Comment:
This looks like it should use OR statement but maybe I don't understand what wrongPath does
```suggestion
        {#if wrongPath || formData.path.length == 0}
```

Reply:
it is `&&`, because when input is empty (mostly when starting app) then error should not be displayed (of course when input is empty then submit button is blocked)

Reply:
```suggestion
        {#if wrongPath && formData.path.length == 0}
```
Ok. Then changing to the length method is all I will suggest here.